projects/1.21/assets/extradelight/extradelight/lang/zh_cn.json

```
@@ -0,0 +1,1610 @@
+  "extradelight.advancement.grater.desc": "用擦丝器整点食材。",
+  "extradelight.advancement.grater.name": "顶刮刮",
+  "extradelight.advancement.grind.desc": "制作或找到研杵。",
+  "extradelight.advancement.grind.name": "粉碎力量",
+  "extradelight.advancement.hellskitchen.desc": "制造一套下界合金餐具。",
+  "extradelight.advancement.hellskitchen.name": "下界厨房",
+  "extradelight.advancement.ingredients.desc": "采集并准备每一种配料。",
+  "extradelight.advancement.ingredients.name": "副主厨",
+  "extradelight.advancement.jelly.desc": "做点果冻！",
+  "extradelight.advancement.jelly.name": "蚂蚁的充气城堡",
+  "extradelight.advancement.jellyall.desc": "制作所有种类的果冻！",
+  "extradelight.advancement.jellyall.name": "遇见彩虹，吃定彩虹",
+  "extradelight.advancement.keg.desc": "制作小桶。",
+  "extradelight.advancement.keg.name": "倒立喝酒！",
+  "extradelight.advancement.lid.desc": "为你的发酵缸制作一个盖子，或者为你的头",
+  "extradelight.advancement.lid.name": "盖上盖子。",
+  "extradelight.advancement.meals.desc": "制作每种可制作的大餐。",
+  "extradelight.advancement.meals.name": "真正的美食家",
+  "extradelight.advancement.melting_pot.desc": "制作熔锅。",
+  "extradelight.advancement.melting_pot.name": "稍微放松一下",
+  "extradelight.advancement.mixingbowl.desc": "做一个搅拌碗。",
+  "extradelight.advancement.mixingbowl.name": "标准的麦片碗大小",
+  "extradelight.advancement.mortar.desc": "制造或找到研磨碗。",
+  "extradelight.advancement.mortar.name": "日常琐事",
+  "extradelight.advancement.noodles.desc": "制作每种可制作的意面。",
+  "extradelight.advancement.noodles.name": "面面面",
+  "extradelight.advancement.oven.desc": "做一个烤炉，解锁更多饭菜！",
+  "extradelight.advancement.oven.name": "热盒子",
+  "extradelight.advancement.pickle.desc": "得到一根腌黄瓜，或者两根",
+  "extradelight.advancement.pickle.name": "嘿嘿嘿！",
+  "extradelight.advancement.sink.desc": "制作水槽",
+  "extradelight.advancement.sink.name": "除了……以外的所有",
+  "extradelight.advancement.snacks.desc": "制作所有的点心。",
+  "extradelight.advancement.snacks.name": "猛炫零食！",
+  "extradelight.advancement.spoon.desc": "做一个勺子，你会需要的。",
+  "extradelight.advancement.spoon.name": "已批准",
+  "extradelight.advancement.start.desc": "小小的额外乐事震撼。",
+  "extradelight.advancement.start.name": "额外乐事",
+  "extradelight.advancement.tap.desc": "做一个水龙头无限接水。",
+  "extradelight.advancement.tap.name": "我会戳这个",
+  "extradelight.advancement.trays.desc": "烤箱的食谱需要烘烤器皿，每个都整一个。",
+  "extradelight.advancement.trays.name": "有价值的容器",
+  "extradelight.advancement.vat.desc": "制作一个发酵缸，掌握发酵的力量！",
+  "extradelight.advancement.vat.name": "一个真正的陶罐……",
+  "extradelight.advancement.vinegarpot.desc": "做一个醋壶，它开始冒泡就可以获得醋。",
+  "extradelight.advancement.vinegarpot.name": "积极腌制",
+  "extradelight.advancement.yeastpot.desc": "做一个酵母罐，它开始冒泡就可以获得酵母。",
+  "extradelight.advancement.yeastpot.name": "小小的肠胃胀气",
+  "extradelight.block.recipefeast.use_container": "还需要其他东西",
+  "extradelight.cactus_juice.hovertext": "这很解渴！",
+  "extradelight.container.chiller.chilled": "已冷却",
+  "extradelight.container.chiller.not_chilled": "未冷却",
+  "extradelight.container.oven": "烤炉",
+  "extradelight.container.oven.served_on": "服务于 %s",

```

```suggestion
  "extradelight.container.oven.served_on": "需盛装于%s",
```
与农夫乐事保持一致
serve此处指上菜